modules/ROOT/pages/prepare-to-install-service-mesh.adoc

```diff
@@ -24,11 +29,16 @@ For more information about how to install Helm, see the Helm installation docume
 ** Azure Kubernetes Service (AKS)
 * Not installed Istio in the cluster
 
-If you want to enable automatic sidecar injection, you must additionally have:
+If you want to enable automatic sidecar injection, you must additionally:
+
 * Install Kubernetes 1.12+ with admissionregistration API 
-*Tip:* To verify the installation, use the command `kubectl api-versions | grep admissionregistration.k8s.io/v1beta1`.
++
+*Tip:* To verify the installation, use the command: 
++
+`kubectl api-versions | grep admissionregistration.k8s.io/v1beta1`.
 * Configure kube-apiserver process with the admission-control flag to “MutatingAdmissionWebhook”
-For more information about how to turn on, see tke Kubernetes documentation.
++
+For more information about how to turn on *turn on what--Tomas?*, see the Kubernetes documentation.
```

Comment:
Incomplete sentence "For more information about how to turn on *turn on what--Tomas?*"

Reply:
Exactly my point. I'll remove it for now.